openedx_webhooks/github_views.py

```diff
@@ -154,7 +154,11 @@ def rescan():
         return jsonify(rescan_repository(repo))
 
     if repo == 'all':
-        repos = get_repos_file().keys()
+        org_url = "https://api.github.com/orgs/edx/repos"
```

Comment:
We'll need to deal with other orgs also somehow.  That was one thing repos.yaml handled for us, since it has a few edx-solutions repos also.